sharktank/tests/models/clip/clip_test.py

```diff
@@ -165,23 +161,31 @@ def runTestCompareIreeAgainstTorchEagerWithInputTokens(
         input_args = OrderedDict([("input_ids", input_ids)])
         batch_size = input_ids.shape[0]
         mlir_path = f"{target_model_path_prefix}.mlir"
-
-        logger.info("Exporting clip text model to MLIR...")
-        export_clip_text_model_iree_test_data(
+        iree_module_path = f"{target_model_path_prefix}.vmfb"
+        export_functions = [
+            ExportFunctionConfig(function=None, batch_sizes=[batch_size])
+        ]
+        target_model = clip_text_model_from_reference_model(
             reference_model=reference_model,
             target_dtype=target_dtype,
-            input_ids=input_ids,
-            target_mlir_output_path=mlir_path,
-            target_iree_parameters_output_path=parameters_path,
+            extra_config_kwargs={
+                "config_path": None,
+                "export_functions": export_functions,
+                "mlir_path": mlir_path,
+                "export_parameters_path": parameters_path,
+                "iree_module_path": iree_module_path,
+                "compile_args": [
+                    "--iree-hal-target-device=hip",
+                    "--iree-hip-target=gfx942",
```

Comment:
This is just moved from elsewhere in the file, but it would be good to get this target from the environment somehow instead of hardcoding it. These tests should be runnable on other GPUs (and CPU, even if slow).